src/DatePicker/DatePicker.js

```diff
@@ -253,6 +320,21 @@ class DatePicker extends Component {
     }
   };
 
+  parseDate = (textDate) => {
```

Comment:
It would be great to allow users using a different date format.
Like `DD-MM-YYYY`.